api.web.yaml

```diff
@@ -138,6 +138,32 @@ paths:
         default:
           description: Unexpected error
 
+  /people/update:
```

Comment:
I think this should be `POST /people/{id}` the 'update'-ness is implied by the method. The `PersonUpdate` then doesn't need the `username` field

Reply:
oh sorry – I should have read the PR description 🤦🏼. I think, if we need to do it, we should do it later because it'll need a cache removal

Reply:
interesting, it hasn't seemed like that locally, but is that because locally cache operates differently?
It immediately booted me out when I updated my admin username and I had to destroy and rebuild it to get back in as an admin

Reply:
ahhh I see. Shall I just disable the edit button for now and we retain the rest of the functionality (though hidden) and follow it up in another PR?

Reply:
I'd remove it – I think there's some chance it'd get re-enabled (potentially by me!) without context. I'm also reasonably confident that we'll not need this functionality